examples/markdown-file/import_map.json

```
@@ -0,0 +1,11 @@
+{
+  "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.0.1/",
```

```suggestion
    "$fresh/": "../../",
```

To use the in-tree version of fresh.